src/featureflagservice/src/ffs_service_bhvr.erl

```diff
@@ -0,0 +1,29 @@
+%%%-------------------------------------------------------------------
+%% @doc Behaviour to implement for grpc service FeatureFlagService.
+%% @end
+%%%-------------------------------------------------------------------
+
+%% this module was generated and should not be modified manually
```

Comment:
How was it generated, can this be moved to the `Dockerfile`

Reply:
Okay that sounds fair, this erlang/elixir stuff is really making me feel out of my element I will admit 😅